supabase/migrations/20230418100618_vector_search.sql

```diff
@@ -0,0 +1,32 @@
+create or replace function vector_search(query_vector vector(1536), match_threshold float, match_count int, min_content_length int)
+returns table (id bigint, input_text text, input_url text, usage_count bigint, similarity float)
+language plpgsql
+as $$
+#variable_conflict use_variable
+begin
+  return query
+  select
+    embedding.id,
+    embedding.input_text,
+    embedding.input_url,
+    embedding.usage_count,
+    (embedding.vector <#> query_vector) * -1 as similarity
```

Comment:
this and line 20 are showing as comments on git, do these work?

Reply:
They do! It's how the dot product is calculated